### PR TITLE
Features/add dynamic ratings trek

### PR DESCRIPTION
**Pull request by JordaneNS**
## Détails de la PR

Ajout des cotations dynamiques (ratings,ratings_description) sur les fiches rando

## Screenshots

image

### CODE REVIEW

File: frontend/src/modules/details/api.ts [hunk 1]
```diff
@@ -9,11 +9,16 @@ const fieldsParams = {
...
+  fields: `${fieldsParams.fields},ratings,ratings_description`,
+  format: 'geojson',
+};
+
 export const fetchDetails = (query: APIQuery, id: string): Promise<RawDetails> => {
   try {
-    return GeotrekAPI.get(`/trek/${id}/`, {
-      params: { ...query, ...fieldsParams, ...portalsFilter },
-    }).then(r => r.data);
+    return GeotrekAPI.get(`/trek/${id}/`, { params: { ...query, ...fieldsParamsDetails } }).then(
```
Comment (dtrucs) [inline frontend/src/modules/details/api.ts:19]: Is there a reason why you deleted `...portalsFilter`?

File: frontend/src/modules/details/api.ts [hunk 2]
```diff
@@ -9,11 +9,16 @@ const fieldsParams = {
   format: 'geojson',
 };
 
+const fieldsParamsDetails = {
+  fields: `${fieldsParams.fields},ratings,ratings_description`,
+  format: 'geojson',
+};
+
```
Comment (dtrucs) [inline frontend/src/modules/details/api.ts:16]: The `fieldsParams` constant doesn't seem to be called anywhere else, why not just add `ratings,ratings_description` to the `fieldsParams` queue?

**Review by dtrucs — CHANGES_REQUESTED** [review]

File: frontend/src/modules/details/api.ts (hunk 1 above)
Comment (JordaneNS) [reply frontend/src/modules/details/api.ts:19]: Oh no ! it's a copy paste error... Thanks for your attention

File: frontend/src/modules/details/api.ts (hunk 2 above)
Comment (JordaneNS) [reply frontend/src/modules/details/api.ts:16]: Yes you're right it's better to add it to the fieldParams queue. Thanks for ur advices :) I fix that !

**Comment by dtrucs** [comment]
This PR looks good to me 👍 
Ok to merge after fixing the CI alerts

**Comment by JordaneNS** [comment]
@dtrucs Hello :)
I fixed the errors of ts but I still have 1 test that don't pass, I don't understand why.

**Comment by dtrucs** [comment]
> @dtrucs Hello :) I fixed the errors of ts but I still have 1 test that doesn't pass, I don't understand why.

You need to create `mock` files for the `trekRating` module. Look at the beginning of the `Details.test.tsx` file to see how the other modules are mocked.

**Pull request merged**